cogs/meme.py

```diff
@@ -52,5 +52,11 @@ async def on_message(self, message):
         elif "hello there" in message.content.lower():
             await message.author.send("https://tenor.com/view/grevious-general-kenobi-star-wars-gif-11406339")
 
+        if "beans" in message.content.lower():
```

Comment:
Use ELIF for consistency with other bot responses, or alternatively change all cases to IF so they can all occur in one message

Reply:
It was to stop spam, i also wanted to keep beans on low priority

Reply:
With an IF statement for this message, you can receive one of the previous messages AS WELL AS the one from this statement. If you want to avoid spam change to an ELIF.